src/preferences/configobject.cpp

```diff
@@ -63,6 +63,11 @@ QString computeResourcePathImpl() {
         else {
             qResourcePath = QCoreApplication::applicationDirPath();
         }
+#elif defined(Q_OS_IOS)
+        // On iOS the bundle contains the resources directly.
+        else {
+            qResourcePath = QCoreApplication::applicationDirPath();
+        }
 #elif defined(__APPLE__)
```

Comment:
```suggestion
#elif defined(Q_OS_MACOS)
```

Reply:
That's fair, in these cases my defensive coding instincts seem to kick in, which is why I like to leave these kinds of fallbacks, but in this case it indeed only makes sense on macOS since I'm not aware of any other Apple OS that uses this bundle structure.